### PR TITLE
First release

### PR DESCRIPTION
**Pull request by mprahl**
Requires some additional feedback before merging

### CODE REVIEW

File: pdc-tangle.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    """
+    The entry point of the tool
+    """
+    # TODO: Should this default change?
+    if not dep_type:
+        dep_type = ['RPMBuildRequires', 'RPMRequires']
+
+    api_url = 'https://{0}/rest_api/v1/'.format(server)
+    client = pdc_client.PDCClient(api_url, develop=True)
+    # TODO: If there are no dependencies, the tree is printed out as the default below. Should this change?
```
Comment (ralphbean) [inline pdc-tangle.py]: Maybe print out a nice "no deps found for artifact X"

If the artifact is absent from the `release-components` endpoint, we should probably print a special message for that.

i.e., there are two cases:
- You wanted to see deps for `whizbang` but PDC has never heard of `whizbang`.
- You wanted to see deps for `whizbang`, and while PDC knows what `whizbang` is, it doesn't know of any deps for `whizbang`.

File: pdc-tangle.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                    ' and RPMRequires.'))
+@click.option('--release', '-r', default='fedora-26',
+              help=('Specifies the release to search under.'
+                    ' This defaults to fedora-26.'))
+@click.argument('artifact')
+def cli(artifact, release, dep_type, server):
+    """
+    The entry point of the tool
+    """
+    # TODO: Should this default change?
```
Comment (ralphbean) [inline pdc-tangle.py]: It works for now.

Can you add the default value to the `@click.option`?

**Comment by mprahl** [comment]
@ralphbean thanks for reviewing. I addressed your two comments. Let me know your thoughts on the rest of the TODO comments.

Also, should I write unit tests for this? Just not sure if the effort is worth it for this particular tool.

**Comment by mprahl** [comment]
Talked with @ralphbean and this is good for an initial release

**Pull request merged**